content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md

```diff
@@ -7,4 +7,77 @@ linkTitle: "Blue-green Deployment"
 weight: 2130
 ---
 
-TBD
\ No newline at end of file
+
+The blue-green release provides a zero downtime deployment, which means the new version can be deployed with the old one preserved. It enables both versions to run at the same time. If there is a problem with running, you can quickly roll back to the old version.
+
+![](./../../../../../static/images/service-mesh/blue-green-0.png)
+
+
+## Before you begin
+
+Firstly, you shoud enable the service-mesh plugin, to make the istio component available.
+
+```
+kubectl edit cc ks-installer -n kubesphere-system
+```
+![](./../../../../../static/images/service-mesh/istio-1.jpg)
+
+
+Secondly, you should set project gateway to enable application governance, in the project advanced settings.
+
+![](./../../../../../static/images/service-mesh/application-1.jpg)
+
+Thirdly, you should have an application, such as bookinfo sample as follows.
+
+![](./../../../../../static/images/service-mesh/application-2.jpg)
```

Comment:
Replace this section with this [doc](https://kubesphere.io/docs/pluggable-components/service-mesh/#enable-service-mesh-before-installation).